requirements-tensorflow-cuda.txt

```diff
@@ -1,5 +1,5 @@
 # Tensorflow with cuda support.
-tf-nightly[and-cuda]==2.16.0.dev20240209  # Pin a working nightly until rc0.
+tensorflow[and-cuda]~=2.16.1  # Pin to TF 2.16
```

Comment:
Should we converge on a standard way to pin jax/torch/tf? Either `~=` on patch version, `>=`, `==`? I can't think of a great reason why each library needs a different approach, and I see all three here.

Reply:
Agree Changed to `>=`